R/nfl.R

```diff
@@ -0,0 +1,31 @@
+# Functions to retrieve NFL information from the Stattleship API
+
+get_nfl_team_info <- function(team_name = "New England") {
+  # Gets NFL team information including nickname, Twitter hashtag and color
+  #
+  # team_name: String of team name or team location. Do not include team nickname.
+  #
+  # e.g. get_nfl_team_info("Denver")
+  
+  # Validation check
+  stopifnot(is.character(team_name))
+  
+  # Get results from Stattleship API
+  results <- ss_get_result(sport = "football", 
+                           league = "nfl",
+                           ep = "teams",
+                           query = list(),
+                           version = 1,
+                           walk = FALSE,
+                           page = NA,
+                           verbose = TRUE)
+  all_team_info <- results[[1]][4][[1]]
```

Comment:
Can R look into the results by key (like "teams") rather than the index?

I don't want to get into a situation where I change the order in the sideloaded entities in the JSON respoone and add one that throws off the order.

But, it's always be` [:teams][:team]` etc.